readme.html

```diff
@@ -0,0 +1,52 @@
+<!-- File: readme.html
+  Copyright (c) 2019-2022 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+either express or implied. See the License for the specific language governing permissions
+and limitations under the License.
+-->
+
+<h3>ePO's Default Permission Set</h3>
+<p>
+  ePO ships with four default permission sets that provide permissions to ePO
+  functionality. The default permission sets available are:
+</p>
+<ul>
+  <li>Executive Reviewer</li>
+  <li>Global Reviewer</li>
+  <li>Group Admin</li>
+  <li>Group Reviewer</li>
+</ul>
+
+<p>
+  For all the actions to execute without any dispute any of the following
+  combinations of permissions will work :
+</p>
+<ul>
+  <li>( Executive Reviewer , Group Admin )</li>
+  <li>( Global Reviewer , Group Admin )</li>
+  <li>( Executive Reviewer , Global Reviewer , Group Admin )</li>
+  <li>
+    ( Executive Reviewer , Global Reviewer , Group Admin , Group Reviewer )
+  </li>
+</ul>
+<hr />
+<p>
+  <strong>
+    <em>Note</em>
+  </strong>
+  :
+  <em>
+    All the permission sets mentioned above are for non-admin users, incase you
+    are an admin you can execute all the actions without any additional
```

Comment:
Please mention what default permissions an admin is having so that in case the customer removes that permissions and the actions don't run, they would be clear on seeing the docs that an admin should have these specific permission sets.

Reply:
Updated the doc for admin permissions